driver-core/src/main/java/com/datastax/driver/core/TableMetadata.java

```diff
@@ -417,15 +434,20 @@ public String exportAsString() {
     @Override
     protected String asCQLQuery(boolean formatted) {
         StringBuilder sb = new StringBuilder();
-        sb.append("CREATE TABLE ").append(Metadata.quoteIfNecessary(keyspace.getName())).append('.').append(Metadata.quoteIfNecessary(name)).append(" (");
+        if(isVirtual()){
+            sb.append("/* VIRTUAL ");
```

Comment:
I know this is extremely unlikely but we could theoretically produce invalid CQL here if the table contains weird column names as in ` create table t1 ("pk */" int primary key)`; this could also happen if some table options contain such forbidden sequences, for example in the `comments` field.
One simple idea to fix this would be to use single-line comments (`--` or `//`) and force `formatted` to false. The only danger would be if we encounter a new line character, but with virtual tables this should never happen.